scripts/deployment/deploy-aave-stop-loss.ts

```diff
@@ -0,0 +1,82 @@
+import { getAdapterNameHash, getCommandHash, getExecuteAdapterNameHash, HardhatUtils } from '../common'
+import hre from 'hardhat'
+import { AAVEAdapter, DPMAdapter, AaveStopLossCommand } from '../../typechain'
+
+async function main() {
+    const utils = new HardhatUtils(hre) // the hardhat network is coalesced to mainnet
+    const signer = hre.ethers.provider.getSigner(0)
+    const network = hre.network.name || ''
+
+    console.log(`Deployer address: ${await signer.getAddress()}`)
+    console.log(`Network: ${network}`)
+    const system = await utils.getDefaultSystem()
+
+    if (
+        utils.addresses.DPM_GUARD === '0x0000000000000000000000000000000000000000' ||
+        utils.addresses.AAVE_POOL === '0x0000000000000000000000000000000000000000' ||
+        !utils.addresses.DPM_GUARD ||
+        !utils.addresses.AAVE_POOL
+    ) {
+        throw new Error('DPM_GUARD and AAVE_POOL must be set in the config file')
+    }
+
+    if (utils.addresses.AAVE_ADAPTER === '0x0000000000000000000000000000000000000000') {
+        console.log('Deploying AAVEAdapter')
+        system.aaveAdapter = (await (
+            await utils.deployContract(hre.ethers.getContractFactory('AAVEAdapter'), [
+                utils.addresses.AUTOMATION_SERVICE_REGISTRY,
+            ])
+        ).deployed()) as AAVEAdapter
+    } else {
+        console.log('Loading AAVEAdapter')
+        system.aaveAdapter = (await hre.ethers.getContractAt(
+            'AAVEAdapter',
+            utils.addresses.AAVE_ADAPTER,
+        )) as AAVEAdapter
+    }
+
+    if (utils.addresses.DPM_ADAPTER === '0x0000000000000000000000000000000000000000') {
+        console.log('Deploying DPMAdapter')
+        system.dpmAdapter = (await (
+            await utils.deployContract(hre.ethers.getContractFactory('DPMAdapter'), [
```

Comment:
shouldn't we do that in separate deploy scripts, for the sake of future updates?

Reply:
by that I mean deployment of the adapters :)